sql/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -616,18 +615,18 @@ private void handleExecute(ByteBuf buffer, Channel channel) {
     }
 
     private void handleSync(final Channel channel) {
+        ReadyForQueryCallback readyForQueryCallback = new ReadyForQueryCallback(channel);
```

Comment:
Is this change here necessary? Seems a bit of a waste to create the object just to have it call `Messages.sendReadyForQuery(channel)` in the non-error case?

Reply:
no it's not, just wanted to streamline it and let it call `sendReadForQuery` at just one place. if you prefer it the other way, I'll revert it.